Dockerfile

```diff
@@ -1,14 +1,17 @@
-FROM golang:1.14
-
-RUN apt-get update && apt-get -y install wget unzip
-
-WORKDIR /tmp
-RUN wget https://releases.hashicorp.com/terraform/0.12.31/terraform_0.12.31_linux_amd64.zip && unzip terraform_0.12.31_linux_amd64.zip -d /usr/local/bin
-
+FROM golang:alpine AS builder
 
 ENV GO111MODULE=on
 WORKDIR /terraform-validator
 COPY . .
-RUN make build
+RUN set -e \
+  && apk --no-cacche --update add make \
+  && make build
+
+FROM alpine:latest
+
+RUN set -e \
+  && apk --no-cacke --update add terraform
```

Comment:
This won't lock terraform to a specific version. Please lock this to a value.

Reply:
Is there a specific terraform version requirement, or lock it to the currently latest one?